packages/bitcore-wallet-client/src/lib/errors/spec.ts

```diff
@@ -49,9 +49,15 @@ var errorSpec = [{
 }, {
   name: 'INSUFFICIENT_FUNDS_FOR_FEE',
   message: 'Insufficient funds for fee.'
+}, {
+  name: 'INSUFFICIENT_ETH_FEE',
+  message: 'Your linked ETH wallet does not have enough ETH for fee.'
 }, {
   name: 'LOCKED_FUNDS',
   message: 'Locked funds.'
+}, {
+  name: 'LOCKED_ETH_FEE',
```

Comment:
we can reuse: https://github.com/bitpay/bitcore/pull/2697/files#diff-e57a24c0f3a56c58bf7457e8e42b712cR56